### PR TITLE
fix: assign ownership of repos to actools

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

**Review by chingor13 — APPROVED** [review]

File: src/teams.json [hunk 1]
```diff
@@ -133,7 +133,15 @@
...
-        "googleapis/sample-tester"
+        "googleapis/sample-tester",
+        "googleapis/google-auth-library-swift",
+        "googleapis/google-api-swift-client",
+        "googleapis/api-common-protos",
+        "googleapis/googleapis.github.io",
+        "googleapis/kiosk",
+        "googleapis/gax-kotlin",
+        "googleapis/disco-to-proto3-converter",
+        "googleapis/api-linter"
```
Comment (noahdietz) [inline src/teams.json:144]: Sorry to be sniping this PR at the last second, but is it possible to assign this to the `api-linter` team? `actools` doesn't actually own this project.

Comment (JustinBeckwith) [reply src/teams.json:144]: No worries at all - this is why I'm leaving it open!  On assigning things to the api-linter team specifically - not quite yet :/  I could create a separate team in sloth for them, give them their own dashboard and such, but it would require them using yoshi live instead of buganizer (for now) as the source of truth.  

Not sure what to do with these quite yet.

Comment (noahdietz) [reply src/teams.json:144]: Ok, no problem, I didn't realize this team assignment was more than just a GitHub team :) Assign away!

Comment (lukesneeringer) [reply src/teams.json:144]: We do not really use Buganizer for linter bugs anyway (we post to GitHub).

**Review by miraleung — APPROVED** [review]

**Review by xiaozhenliu-gg5 — COMMENTED** [review]
Hi Justin, can we add https://github.com/googleapis/proto-breaking-change-detector? This is the tool wee integrate with the presubmit service and check breaking changes for protos. Thank you!

File: src/teams.json [hunk 2]
```diff
@@ -133,7 +133,15 @@
         "googleapis/gapic-generator",
         "googleapis/nodejs-proto-files",
         "googleapis/proto-plus-python",
-        "googleapis/sample-tester"
+        "googleapis/sample-tester",
```
Comment (miraleung) [inline src/teams.json:136]: Add `api-common-java`?

Not part of this PR, but consider sorting this list alphabetically.
